### PR TITLE
[WIP] Pipeline integration testing using helm & minikube (SHOULD FAIL CI)

### PR DESCRIPTION
**Pull request by lee0c**
Work toward #33 #108 

## Proposed Changes

  - Add an additional pipeline job to spin up a cluster & deploy promitor via helm chart
  - [To do] Test that promitor is live
  - [To do] Test that promitor is providing the specified metrics

### CODE REVIEW

**Comment by tomkerkhove** [comment]
@lee0c 	are you up for finishing this or do you prefer to hand this over to somebody?

**Comment by lee0c** [comment]
@tomkerkhove I was actually about to message. I'm up to finish this or at least put more work to it but I'm limited in what I can do without having access to the pipeline, so if you can make sure the pipeline is triggering properly on these PRs that would help me get some changes made. Additionally it'd be helpful to chat with you at some point in the week about how to handle secrets & metric declaration details in the pipeline. I'm on eastern time this week so we're only 6 hours apart.

**Comment by tomkerkhove** [comment]
@lee0c feel free to drop me an email on kerkhove.tom@gmail.com to schedule something next week!

**Comment by lee0c** [comment]
@tomkerkhove Minikube is working in the pipeline now. I'm ironing out some issues w/ helm init & making sure the basic deployment/debug output is working as intended, but after that this should be good for you to add more detailed integration testing.

**Comment by tomkerkhove** [comment]
Thanks for the update and the great progress @lee0c!

File: .azure-devops/scraper-ci.yml [hunk 1]
```diff
@@ -72,7 +73,7 @@ jobs:
       $buildNumber = '$(Build.BuildNumber)'
       Write-Host "Initial build number: '$buildNumber'"
       Write-Host "PR Number '$prNumber'"
-      $buildNumber = 'PR$(System.PullRequest.PullRequestNumber)'
+      $buildNumber = 'pr$(System.PullRequest.PullRequestNumber)'
```
Comment (tomkerkhove) [inline .azure-devops/scraper-ci.yml]: What's the reason for this rename if I may ask?

**Comment by tomkerkhove** [comment]
Hi @lee0c 	would you be up for finishing this over summer?

**Comment by lee0c** [comment]
@tomkerkhove Oh yes! I am so sorry, I've been crazy busy. This is pretty close to working from what I remember, I was trying to iron out an issue with Minikube. I'll update this soon!

**Comment by tomkerkhove** [comment]
No problem @lee0c 	and there is no rush, I was just wondering if you were up for it or if I should take over.

Given you are busy at the moment I'd suggest to hold off until v1.0 is shipped (should be July) given we have breaking change on configuration side (#431) and multiple resources per metrics (#513)

Then I can jump in to help where help is needed, would that help you?

**Comment by tomkerkhove** [comment]
Ok if we abandon this one @lee0c ?

**Comment by lee0c** [comment]
@tomkerkhove Yes, apologies for never picking this up again 😢 Work got busy and moved in other directions.

**Comment by tomkerkhove** [comment]
No worries @lee0c!

**Pull request closed without merging**